src/coreclr/src/vm/threadsuspend.cpp

```diff
@@ -404,26 +389,18 @@ Thread::SuspendThreadResult Thread::SuspendThread(BOOL fOneTryOnly, DWORD *pdwSu
         }
 #endif // _DEBUG
 
-        if (doSwitchToThread)
```

Comment:
Most of this code is simpler because fibers stuff was removed. For example without fibers `doSwitchToThread` would be always `TRUE` so no longer needed, and so on..